docusearch/management/commands/import_documents.py

```diff
@@ -121,11 +122,21 @@ def process_agency(documents_directory, agency):
             process_office(agency_directory, agency, d)
 
 
+def _get_documents_directory():
+    """ Returns the default document directory """
+
+    return os.path.join(
+        BASE_DIR.replace('/foia-hub/foia_hub', ''), 'foia_docs')
+
+
 class Command(BaseCommand):
 
     def handle(self, *args, **options):
-        documents_directory = '/vagrant/data/responsive'
-        agencies = os.listdir(documents_directory)
+        try:
```

Comment:
This is a bit more complicated than it needs to be. I just did a simpler (and more user-friendly) version here: https://github.com/projectkrang/foia-hub/pull/6/files#diff-d54d215e00dd197ce44f34a51e05d983R128

I think you could close this pull request. 

All the BASE_DIR stuff is unnecessarily complicated.